Library/Homebrew/extend/ENV/super.rb

```diff
@@ -29,6 +29,13 @@ def self.extended(base)
     base.run_time_deps = []
   end
 
+  # The location of Homebrew's shims on this OS.
+  # @public
+  sig { returns(Pathname)) }
+  def shims_path
+    HOMEBREW_SHIMS_PATH
+  end
```

Comment:
There should probably be an declaration in either the shared or std env, since we do that for every other ENV method.

Reply:
> There should probably be an declaration in either the shared or std env, since we do that for every other ENV method.

@Bo98 I disagree. This is specific to superenv so I'd intend it to be used only with `Superenv.shims_path`.

> Technically the generic OS case should probably be `HOMEBREW_SHIMS_PATH/"super"`, so I suggest maybe doing

👍🏻

Reply:
> I'd intend it to be used only with `Superenv.shims_path`.

In that case that's fine. I was thinking `ENV.` at first.